lib/nerves/artifact/build_runners/docker.ex

```diff
@@ -125,7 +125,14 @@ defmodule Nerves.Artifact.BuildRunners.Docker do
 
     mounts = Enum.join(mounts(pkg), " ")
     ssh_agent = Enum.join(ssh_agent(), " ")
-    cmd = "docker run --rm -it -w #{@working_dir} #{mounts} #{ssh_agent} #{image}"
+
+    cmd =
+      "docker run --rm -it --user #{uid()}:#{gid()} -w #{@working_dir} #{mounts} #{ssh_agent} #{
+        image
+      }"
+      |> IO.inspect()
```

Comment:
Do you want to leave in the IO.inspect?

Reply:
FWIW i really like Credo, and it can lint/catch that